Libs/MRML/Core/vtkMRMLSelectionNode.cxx

```diff
@@ -149,6 +156,7 @@ void vtkMRMLSelectionNode::WriteXML(ostream& of, int nIndent)
   of << " activeTableID=\"" << (this->ActiveTableID ? this->ActiveTableID : "NULL") << "\"";
   of << " activeViewID=\"" << (this->ActiveViewID ? this->ActiveViewID : "NULL") << "\"";
   of << " activeLayoutID=\"" << (this->ActiveLayoutID ? this->ActiveLayoutID : "NULL") << "\"";
+  of << " activePlotLayoutID=\"" << (this->ActivePlotLayoutID ? this->ActivePlotLayoutID : "NULL") << "\"";
```

Comment:
here I mean the vtkMRMLPlotLayoutNode. I think just calling Plot can be misleading to a vtkMRMLPlotNode, no?